### PR TITLE
Updated to LÖVE 11.1

### PR DESCRIPTION
**Pull request by suttonwilliamd**
With the latest version of LÖVE, colors were changed to 0-1 range instead of 0-255.  This commit makes BrainLOVE visible again!

### CODE REVIEW

File: main.moon [hunk 1]
```diff
@@ -19,37 +19,37 @@ love.load = () ->
...
+				resetButton\setColor 1, 0.588, 0.588
 
 			running = not running
 	))
 
 	runButton\onHover(((entered) =>
 		if entered
 			if running
-				runButton\setColor 125, 200, 125
+				runButton\setColor 0.49, 200, 0.49
```
Comment (BytewaveMLP) [inline main.moon]: `200`  should be `0.78`, keeping with the trend here

File: main.moon [hunk 2]
```diff
@@ -19,37 +19,37 @@ love.load = () ->
...
 			else
-				runButton\setColor 200, 125, 125
+				runButton\setColor 200, 0.49, 0.49
```
Comment (BytewaveMLP) [inline main.moon]: Here too

**Review by BytewaveMLP — CHANGES_REQUESTED** [review]
Missed a couple of 0-255 -> 0-1 scale conversions, but otherwise LGTM.

(kind of annoyed that LÖVE changed this in their stdlib, but maybe it was better for color math or something)

**Comment by suttonwilliamd** [comment]
I wasn't sure how many decimal places were useful with the color data.  My math says 175 should be 0.686~ I only used 0.49 for 125 because it came out to 0.49019~.  My question is, should I change my 0.686's to just 0.68's while I'm fixing this thing you pointed out, or change the 200's to 0.784?

Thanks!

**Comment by BytewaveMLP** [comment]
My math training would tell me to use `0.490`, just to make it clear we're working at 3 decimal places everywhere.

Otherwise, LGTM.

~~I really ought to get back and finish this project at some point, but text input has had me stuck for a while. :p~~

**Review by BytewaveMLP — APPROVED** [review]

**Pull request merged**